src/towncrier/test/test_build.py

```diff
@@ -500,54 +501,92 @@ def test_no_package_changelog(self):
             ).lstrip(),
         )
 
-    def test_single_file(self):
+    def test_single_file_false(self):
         """
-        Enabling the single file mode will write the changelog to a filename
-        that is formatted from the filename args.
+        Disabling the single file mode will write the changelog to a filename
```

Comment:
I think that the docstring of the tests should be self contained and not link to the documentation.

It should explain the targeted feature, regardless of the documentation.

In some way, the docstring of  this test serves as the "real" documentation :)